mylyn.tasks/connectors/gitlab/org.eclipse.mylyn.gitlab.core/src/org/eclipse/mylyn/gitlab/core/GitlabTaskCommentMapper.java

```diff
@@ -0,0 +1,7 @@
+package org.eclipse.mylyn.gitlab.core;
```

Comment:
We need to put more work here before promoting this class to API, I suggest to move to `internal` package until it will be ready.